src/components/Dialogs/ImageCropDialog/ImageCropDialog.style.ts

```diff
@@ -14,6 +14,9 @@ const roundedCropperCss = css`
     border-radius: 50%;
   }
 `
+export const StyledZoomButton = styled(Button)`
+  cursor: pointer;
```

Comment:
Doesn't button already have this style?

Reply:
nope. Maybe we should update the Button as well at some point

Reply:
Wait, my mistake, you were right.  I just found that if you add `onClick`, `cursor` will be added